examples/flutter_gallery/android/fastlane/README.md

```diff
@@ -0,0 +1,29 @@
+fastlane documentation
+================
```

Comment:
Could you please add one sentence explaining what we use fastlane for so that a reader does not need to google it? Something like "We use fastlane for automating building APKs and pushing then to Google Play alpha channel." Then a user who's not interested in the topic would move on immediately.

Reply:
ha, I was gonna reply how frustrating it is that this file gets continuously autogen-ed, then found this https://docs.fastlane.tools/actions/skip_docs/

Reply:
Done